Document/0x05i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -268,23 +255,67 @@ Add the following to build.gradle:
 ### Testing for Debugging Code and Verbose Error Logging
 
 #### Overview
+StrictMode is a developer tool able to detect policy violation, e.g. disk or network access.
+It could be implemented in order to check the use of good coding practices such as no slow code or network access on the main thread.
+The policy are defined together with rules and different methods of showing the violation of a policy.
+
+There are two category of policies:
+* `StrictMode.ThreadPolicy`
+* `StrictMode.VmPolicy`
+
+The ThreadPolicy can monitor:
+* Disk Reads
+* Disk Writes
+* Network access
+* Custom Slow Code
+
+The VM policies,  applied to all threads in the virtual machine's process, are:
```

Comment:
Same as above. I would remove this and put a reference after "StictMode.VmPolicy" to link to the official documentation.